src/core/client/admin/components/UserHistoryDrawer/UserDrawerNotesContainer.css

```diff
@@ -0,0 +1,30 @@
+.root {
+  
+}
+
+.textArea {
+  width: 100%;
+  box-sizing: border-box;
+  height: calc(12 * var(--mini-unit));
+  border-color: var(--palette-grey-main);
+  padding: calc(0.5 * var(--mini-unit));
+  border-radius: 2px;
+  margin-bottom: var(--spacing-2);
+  padding: var(--spacing-3);
+  font-weight: var(--font-weight-regular);
+  font-family: var(--font-family-sans-serif);
+  font-size: calc(16rem / var(--rem-base));
+  line-height: 1;
+  letter-spacing: calc(0.2em / 16);
+  color: var(--palette-text-primary);
```

Comment:
Firefox adds a `border-width: 2px;` when unspecified, perhaps add a `border-width: 1px;` to have it match chrome?